trunk/SUAVE/Methods/Weights/Correlations/General_Aviation/empty.py

```diff
@@ -0,0 +1,314 @@
+    output.landing_gear      = Data()
+    output.landing_gear_main = wt_landing_gear.main
+    output.landing_gear_nose = wt_landing_gear.nose
+    output.horizontal_tail   = wt_tail_horizontal
+    output.vertical_tail     = output_3.wt_tail_vertical
+    
+    output.systems           = output_2.wt_systems       
+    output.systems_breakdown = Data()
+    output.systems_breakdown.control_systems   = output_2.wt_flt_ctrl    
+    #output.systems_breakdown.apu               = output_2.wt_apu         
+    output.systems_breakdown.hydraulics        = output_2.wt_hyd_pnu     
+    output.systems_breakdown.avionics          = output_2.wt_avionics    
+    output.systems_breakdown.electrical        = output_2.wt_elec        
+    output.systems_breakdown.air_conditioner   = output_2.wt_ac          
+    output.systems_breakdown.furnish           = output_2.wt_furnish    
+    output.systems_breakdown.fuel_system       = output_2.wt_fuel_sys
+    #define weights components
+
+    
+    
+    control_systems   = SUAVE.Components.Physical_Component()
+    electrical_systems= SUAVE.Components.Physical_Component()
+    passengers        = SUAVE.Components.Physical_Component()
+    furnishings       = SUAVE.Components.Physical_Component()
+  
+    apu               = SUAVE.Components.Physical_Component()
+    hydraulics        = SUAVE.Components.Physical_Component()
+   
+    
+    
+    #assign output weights to objects
+    control_systems.mass_properties.mass                             = output.systems_breakdown.control_systems
+    electrical_systems.mass_properties.mass                          = output.systems_breakdown.electrical
+    passengers.mass_properties.mass                                  = output.pax + output.bag
+    furnishings.mass_properties.mass                                 = output.systems_breakdown.furnish
+    avionics.mass_properties.mass                                    = output.systems_breakdown.avionics              
+   
+    #fuel.mass_properties.mass                                        = output.fuel
+    hydraulics.mass_properties.mass                                  = output.systems_breakdown.hydraulics
+    
+    if has_air_conditioner:
+        vehicle.air_conditioner.mass_properties.mass                 = output.systems_breakdown.air_conditioner
+    #assign components to vehicle
+    vehicle.control_systems                     = control_systems
+    vehicle.electrical_systems                  = electrical_systems
+    vehicle.avionics                            = avionics
+    vehicle.furnishings                         = furnishings
+    vehicle.passenger_weights                   = passengers 
+   
+    vehicle.apu                                 = apu
+    vehicle.hydraulics                          = hydraulics
+    vehicle.landing_gear                        = landing_gear_component
+
+    #note; air conditioner optional, and weight is added to the air_conditioner object directly
```

Comment:
I think it's okay to make it optional, just change the key dependence.

Reply:
done